frontend/src/scenes/project-homepage/recentInsightsLogic.ts

```diff
@@ -0,0 +1,32 @@
+import { kea } from 'kea'
+
+import { teamLogic } from 'scenes/teamLogic'
+import { InsightModel } from '~/types'
+import api from 'lib/api'
+
+import type { recentInsightsLogicType } from './recentInsightsLogicType'
+export const recentInsightsLogic = kea<recentInsightsLogicType>({
+    path: ['scenes', 'project-homepage', 'recentInsightsLogic'],
+    connect: {
+        values: [teamLogic, ['currentTeamId']],
+    },
+    loaders: ({ values }) => ({
+        recentInsights: [
+            [] as InsightModel[],
+            {
+                loadRecentInsights: async () => {
+                    const response = await api.get(
+                        `api/projects/${values.currentTeamId}/insights/?my_last_viewed=true&order=-my_last_viewed_at`
+                    )
+                    return response.results
+                },
+            },
+        ],
+    }),
+
+    events: ({ actions }) => ({
+        afterMount: () => {
+            actions.loadRecentInsights()
```

Comment:
If it's not too much of a hassle, it'd be icing on the cake if we have a kea logic test for this.

Reply:
Just moved this into the `projectHomepageLogic` and added a test!